build/nuget/Uno.WinUI.MSAL.nuspec

```
@@ -13,58 +13,58 @@
 		<dependencies>
 			<group targetFramework="net9.0">
 				<dependency id="Uno.WinUI" version="to-be-set-by-ci" />
-				<dependency id="Microsoft.Identity.Client" version="4.61.3" />
+				<dependency id="Microsoft.Identity.Client" version="4.73.1" />
```

The version containing the fix is `4.72.1`, not `4.73.1`.


```suggestion
				<dependency id="Microsoft.Identity.Client" version="4.72.1" />
```

---

Yes, let's keep the oldest possible.

---

Thank you for the details :) !